Sources/swift-doc/Subcommands/Generate.swift

```diff
@@ -47,6 +51,8 @@ extension SwiftDoc {
     var options: Options
 
     func run() throws {
+     let datesLocale = Locale(identifier: self.options.datesLocale)
```

Comment:
I don't think the codebase uses `self.` where it isn't needed, so I suggest omitting it here for consistency.

```suggestion
     let datesLocale = Locale(identifier: options.datesLocale)
```

Reply:
Good call, thank you!
I started with `let datesLocale = Locale(identifier: self.datesLocale)`, because I forgot it was inside options, and then forgot to remove the self 😅